controllers/comments.js

```diff
@@ -285,5 +327,38 @@ export default {
         error: e.message
       });
     }
+  },
+
+  /**
+   * This method gets all the highlighted texts on the article
+   * @param {Object} req
+   * @param {Object} res
+   * @returns {Object} Object of highlighted texts on the article
+   */
+  getHighlighted: async (req, res) => {
+    const { slug } = req.params;
+
+    // check if the article with slug provided is in db
+    const findArticle = await article.findOne({
+      where: { slug }
+    });
+    if (!findArticle) {
+      return res.status(404).send({
+        status: 404,
+        errorMessage: 'Article not found'
```

Comment:
We decided with the team that we can return error not error message in case an error occurs, it's better if you refactor this.